<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RingDialog</class>
 <widget class="QDialog" name="RingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>275</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Door ring detected</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(143, 240, 164)</string>
  </property>
  <widget class="QPushButton" name="cmdOpenDoor">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>210</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Open house door</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblBell">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>[img_bell]</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblMessage">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>581</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Ubuntu&quot;; color: rgb(61, 56, 70)</string>
   </property>
   <property name="text">
    <string>Door bell rings!</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
